resources/guidelines/testing/store/README.md

```diff
@@ -1,11 +1,72 @@
+
+    * Is a general function as described in your extension available?
+    * Do the buttons, export, rules, etc., work?
+    * Are errors displayed in the console?
+
+* **[Lighthouse audit home/listing/detail](../store/quality-guidelines-apps/README.md#frontend-apps)** - We check:
+
+    * If your extension affects the storefront or not?  (so that the search engines have no problems with it).
+    * If all buttons, labels, etc., are named correctly?
+
+We pay attention to all five audits. The app must not limit these. Like most search engines, we also pay attention to mobile-first.
+
+* **[Rich snippet home/listing/detail](../store/quality-guidelines-apps/README.md#template-tests)** - We check:
+
+    * If the page can be indexed?
+    * Is there any incorrect price information being displayed?
+
+Rich snippets have no influence on the ranking of a website. Thus, they do not count among the ranking factors. Nevertheless, search hits enriched with additional information have various SEO advantages: higher attention, higher click-through rate, and greater relevance.
+
+* **[No errors in the storefront and 503/404 errors](../store/quality-guidelines-apps/README.md#error-messages-must-be-entered-in-the-event-log)** - We check:
+
+    * If the app is active in the storefront?
+    * If it involves display errors and errors of any kind?
+
+The end customer should not receive any misleading error messages. It does not matter whether a function causes the error or the customer does not use the function correctly. For example, the customer can upload a picture using a function, but if the customer tries to upload a video, a clear message should be displayed here.
+
+* **[Cookie check storefront/checkout](../store/quality-guidelines-apps/README.md#register-a-cookie-to-the-cookie-consent-manager)** - Since the GDPR/DSGVO, the classification of cookies is particularly important. We distinguish between three types of cookies.
+
+    * **Technically required**: Only cookies that are really important for the store without which no purchase would be possible.
+
+    * **Comfort functions**: Cookies to display personalized ads as banners, newsletter pop-ups, and content from video and social media platforms.
+
+    * **Statistics and Tracking**: Statistics and everything that has to do with data collection and tracking.
+
+* **[Store description German/English](../store/quality-guidelines-apps/README.md#app-descriptions-in-your-shopware-account)** - The app store description includes several points if the app can be used only in a specific country, so leave this clearly in the description. The German description is only mandatory if the app is to be offered in the German market. Furthermore, there must always be at least two images of the app in English, e.g., of the storefront and the admin.
```

Comment:
The manufacturer profile is still missing here. It must also be filled before the item can be marked as OK.
![manufacturer](https://user-images.githubusercontent.com/119946268/211496085-7fbed9ee-4198-4a6b-95eb-062be12824da.png)

Reply:
This info is already covered in the redirect mentioned. However, let me know if there should be a separate point mentioned about it. If yes, then I request you to share related details like what is checked in the manufacturer profile before it is marked OK.

Reply:
Oh ok, sorry I have not seen it. Then it is fine for me.